### ISSUE TITLE
Brave browser: NVDA doesn't read contextual menu to fill user/password

### ISSUE THREAD

**Issue by iubito** [nvaccess/nvda#20401]
### Brief summary

I don't know if it is a Brave or NVDA issue, as it appears after both updates recently. Tested on current version and beta version of NVDA.

When logging in a website, a form ask for username and password. Saved user/pass are shown in a contextual menu when I press the arrow down key.

Before the bug appeared, NVDA said "password for MyUserName1", "password for myUserName2".
Since the bug appeared, NVDA say the content of the input field, "empty" or the beginning of typed username.

### Steps to reproduce

1. Using Brave browser, go to any user/password form, for example https://en.wikipedia.org/wiki/Special:Userlogin
2. Log in at least one time, save login informations.
3. Log out
4. Go back to the log in page, if you click the "username" field or use down arrow key, a menu appear, showing your saved username(s), and an item to manage saved usernames.
5. Use down arrow key or the mouse pointer to choose an item in this menu, NVDA says "empty" ("vide" in French) if the field is empty, or the beginning of the usernamed your typed, but nothing from the menu.

### Actual behavior

NVDA says the content of the login/username field, instead of the menu to choose a saved username.

[nvda.log](https://github.com/user-attachments/files/29316898/nvda.log)

### Expected behavior

As it worked before Brave and NVDA update, the content of the menu was vocalized.

### NVDA logs, crash dumps and other attachments

[nvda.log](https://github.com/user-attachments/files/29316912/nvda.log)

### NVDA type

installed copy

### NVDA version

2025.3.3.54605

### Have you tried any other versions of NVDA? If so, please report their behaviors.

I tried beta nvda_snapshot_beta-56638,e1e0db9c.exe, not installed, run from from the installer.

### Windows version

10.0.19045 Build 19045

### Name and version of other software in use when reproducing the issue

[Brave 1.91.175 (Build officiel) (64 bits)](https://brave.com/latest/)
Chromium: 149.0.7827.155

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have not restarted NVDA with add-ons disabled

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have run the System Accessibility Repair Tool and the issue still occurs

**Comment by Matstn** [comment]
I experience the same issue with Google Chrome and all text inputs with value suggestions

**Comment by seanbudd** [comment]
Related #13522

**Comment by seanbudd** [comment]
Please report this to Chromium
